webpack/entry.js

```diff
@@ -1,6 +1,5 @@
 import './animateThunder'
 import './animateTime'
```

Comment:
Shouldn't you delete this instead of `'./animateHeader'`?

Reply:
(this == `import './animateTime'`)

Reply:
whoops. Good riddance. Almost broke the entire JS of the page.